docker/emptydrops/npz2rds/npz2txt.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+## Author: Nick Barkas <nbarkas@broadinstitute.org>
```

Comment:
nit: use """ than ## in python scripts for docstrings, which can be parsed by tools when it's needed

Reply:
Please provide an example

Reply:
It's basically use triple quotes instead of # for the module-level docstrings, but since it's trivial, you don't have to make this change at this moment, we shall use a linter once we finish optimus and change all places together. FYI: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
ok